packages/manager/modules/pci/src/projects/new/config/index.html

```diff
@@ -71,4 +71,28 @@ <h1 data-translate="pci_project_new_config_title"></h1>
             <span data-translate="pci_project_new_config_btn_return_to_projects"></span>
         </a>
     </div>
+
+    <!--Banner create project info-->
+    <div
+        data-ng-if="$ctrl.supportTicketLink"
+    >
+        <span class="mx-1 oui-icon oui-icon-info-circle"></span>
+        <span
+            class="mb-2 font-weight-bold"
+            data-translate="pci_project_new_config_create_project_title"
+        ></span>
+        <span
+            class="d-block text-justify"
+            data-translate="pci_project_new_config_create_project_info"
+        ></span>
+        <a
+            class="d-block"
+            data-ng-href="{{:: $ctrl.supportTicketLink}}"
+            target="_blank"
+            rel="noopener"
+        >
+            <span data-translate="pci_project_new_config_create_project_info_link"></span>
+            <span aria-hidden="true"></span>
```

Comment:
Are you displaying some icon here or is there another reason to use `aria-hidden` here?

Reply:
Ah no, i have to remove it, we had one but we removed it.
Forgot to delete the accessibility related span.

Thanks.